PRIVACY.md

```diff
@@ -55,12 +64,22 @@ data is transferred to us without legal permission or consent.
 4.1. Newsletters and emails that are not service-related, are not fulfilling
 our duties to perform our service and contain advertisement will only be send
 to you, if you have given us your prior consent.
-
-4.2. The content of the newsletter or email follows from its respective
-description. If no description is available, the content sent comprises
-information about us and our services.
-
-4.3. If you no longer want to receive our newsletter you can opt to stop
```

Comment:
Description of how we document your consent using double optin and storing IP address of optin request.